front-end-web/src/Components/Layouts/NavBar.tsx

```diff
@@ -1,21 +1,102 @@
 import React, { Fragment } from "react";
 import { Props } from "../../@Types";
+import { createStyles, makeStyles, Theme } from "@material-ui/core/styles";
+import AppBar from "@material-ui/core/AppBar";
+import Toolbar from "@material-ui/core/Toolbar";
+import Typography from "@material-ui/core/Typography";
+import IconButton from "@material-ui/core/IconButton";
+import MenuIcon from "@material-ui/icons/Menu";
+import AccountCircle from "@material-ui/icons/AccountCircle";
+import MenuItem from "@material-ui/core/MenuItem";
+import Menu from "@material-ui/core/Menu";
 
 /**
  * TODO: Add Documentation
  */
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      flexGrow: 1
+    },
+    menuButton: {
+      marginRight: theme.spacing(2)
+    },
+    title: {
+      flexGrow: 1
+    }
+  })
+);
+
 const NavBar: React.FunctionComponent<Props> = ({
   state,
   handlers,
   classes
 }) => {
+  const [auth, setAuth] = React.useState(true);
+  const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
+  const open = Boolean(anchorEl);
+
+  const handleMenu = (event: React.MouseEvent<HTMLElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
   return (
-    <Fragment>
-      {
-        //TODO
-      }
-    </Fragment>
+    <div className={classes.root}>
```

Comment:
What you have here is all correct in terms of following best practices, but for sake of the demo can you switch this from `classes.root` to `classes2.root`? We'll need to switch it back in the future, but for the time being this will be faster to set up for the demo.